tests/test_humidity_sensor.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+import unittest
+
+import mock
+
+from greenpithumb import humidity_sensor
+from greenpithumb import dht11_exceptions
+
+class HumiditySensorTest(unittest.TestCase):
+
+	def setUp(self):
+		self.mock_dht11_result = mock.Mock()
+		self.humidity_sensor = humidity_sensor.HumiditySensor(
+			self.mock_dht11_result)
+
+	def test_humidity_50(self):
+		"""Value given should be value returned."""
+
+		self.mock_dht11_result.error_code.return_value = 0
```

Comment:
I think this might be a moo point if we change to taking a `DHT11` instance instead of a result, but for mocking simple attributes (as opposed to functions), I think we can just do

```python
self.mock_dht11_result.error_code = 0
```